test/run

```diff
@@ -12,6 +12,12 @@ begin "$root/test_basic"
 	assert_missing "./expected.json"
 end
 
+begin "$root/test_wrong_config"
+	faucet || echo "Crashed successfully"
+	# TODO: More assertions
+	mkdir "./dist"
```

Comment:
This is necessary, because otherwise the thing crashes when it tries to remove the dist folder...